crates/router/Cargo.toml

```diff
@@ -59,6 +59,7 @@ blake3 = "1.8.2"
 bytes = "1.10.1"
 clap = { version = "4.5.38", default-features = false, features = ["std", "derive", "help", "usage"] }
 config = { version = "0.14.1", features = ["toml"] }
+prost-types = "0.13"
```

Comment:
Can you make this dependency optional and enable it when the necessary feature is enabled?